src/runtime/GlobalObjectBuiltinIntl.cpp

```diff
@@ -2560,6 +2561,9 @@ void GlobalObject::installIntl(ExecutionState& state)
     m_intlCollator->getFunctionPrototype(state).asObject()->defineOwnProperty(state, state.context()->staticStrings().resolvedOptions,
                                                                               ObjectPropertyDescriptor(new FunctionObject(state, NativeFunctionInfo(strings->resolvedOptions, builtinIntlCollatorResolvedOptions, 0, nullptr, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::ConfigurablePresent | ObjectPropertyDescriptor::WritablePresent)));
 
+    m_intlCollator->getFunctionPrototype(state).asObject()->defineOwnProperty(state, ObjectPropertyName(state, state.context()->vmInstance()->globalSymbols().toStringTag),
+                                                                              ObjectPropertyDescriptor(strings->Object.string(), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::NotPresent)));
```

Comment:
Could you check the spec? 
`Intl.Collator.prototype [ @@toStringTag ]` has configurable attribute according to the following spec.
> This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.

Reply:
True, I misread the spec, thanks.